ray-operator/apis/ray/v1alpha1/rayservice_types.go

```diff
@@ -59,10 +61,22 @@ type RayActorOptionSpec struct {
 // RayServiceStatus defines the observed state of RayService
 type RayServiceStatus struct {
 	// Important: Run "make" to regenerate code after modifying this file
-	ServiceStatus    ServiceStatus           `json:"serviceStatus,omitempty"`
-	ServeStatuses    []ServeDeploymentStatus `json:"serveDeploymentStatuses,omitempty"`
-	RayClusterName   string                  `json:"rayClusterName,omitempty"`
-	RayClusterStatus RayClusterStatus        `json:"rayClusterStatus,omitempty"`
+	ServiceStatus        ServiceStatus           `json:"serviceStatus,omitempty"`
```

Comment:
question:
is there a way to document these so they show up in the CRD description as well? I want to carefully define the status names and semantics.

Reply:
You can just add a comment above each member and KubeBuilder will turn it into a description in the CRD.
However, I think we have an option somewhere that clips the description to 100 characters...

Reading the generated CRDs is not a good experience, though -- at some point we should figure out how to auto-generate docs from the CRDs. I think many projects do this.